weave-js/src/core/ops/domain/run.ts

```diff
@@ -689,3 +689,30 @@ export const opRunRuntime = makeRunOp({
     return run.computeSeconds;
   },
 });
+
+export const opStrRunlink = OpKinds.makeBasicOp({
+  name: 'str-runlink',
```

Comment:
please mark this has hidden - else it will show up as a recommended op off of strings